app/src/main/java/org/simple/clinic/patient/PatientConfig.kt

```diff
@@ -1,20 +1,47 @@
 package org.simple.clinic.patient
 
-/**
- * See the [SQLite Limits](https://www.sqlite.org/limits.html) doc for more information.
- *
- * See the section "maximum number of host parameters in a single SQL statement".
- * */
-const val MAXIMUM_SQLITE_QUERY_LIMIT = 1000
+import androidx.annotation.VisibleForTesting
+import io.reactivex.Observable
+import org.simple.clinic.remoteconfig.ConfigReader
 
 data class PatientConfig(
     val limitOfSearchResults: Int,
     val scanSimpleCardFeatureEnabled: Boolean,
     val recentPatientLimit: Int
 ) {
-  init {
-    if (limitOfSearchResults !in 1 until MAXIMUM_SQLITE_QUERY_LIMIT) {
-      throw AssertionError("limit of search results must be within [1, 999]")
+
+  companion object {
+
+    /**
+     * See the [SQLite Limits](https://www.sqlite.org/limits.html) doc for more information.
+     *
+     * See the section "maximum number of host parameters in a single SQL statement".
+     * */
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    const val MAXIMUM_SQLITE_QUERY_LIMIT = 1000L
+
+    fun read(configReader: ConfigReader): Observable<PatientConfig> {
+      return Observable.fromCallable {
+        val numberOfSearchResults = configReader
+            .long("patients_numberofsearchresults", 100L)
+            .coerceAtLeast(1L)
+            .coerceAtMost(MAXIMUM_SQLITE_QUERY_LIMIT)
+            .toInt()
+
+        val bpPassportFeatureEnabled = configReader
+            .boolean("patients_bppassportfeature_enabled", false)
+
+        val numberOfRecentPatients = configReader
+            .long("patients_recentpatientsinhomescreen", 10L)
```

Comment:
The name for config here does not tell me what this number is?
So I was thinking that now that we have started using Remote configs, we should define a naming convention.
How about something like `<configType>_<configName>_<configUnitDesc>`. Eg: `patients_searchresults_limit`. 
Wdyt?
So you can you make this as `patients_recentpatients_limit`?

Reply:
Makes sense, I'll do this.

Reply:
Done.